resources/qml/MatrixText.qml

```diff
@@ -29,8 +31,32 @@ TextEdit {
         id: ma
 
         anchors.fill: parent
-        acceptedButtons: Qt.NoButton
+        acceptedButtons: {
+            if (message.selectedText == "")
+                return Qt.NoButton;
+            else
+                return Qt.RightButton;
+        }
+
         cursorShape: parent.hoveredLink ? Qt.PointingHandCursor : Qt.ArrowCursor
+
+        onClicked: {
+            if (parent.selectedText != "")
+                copyMenu.popup();
+        }
+    }
+
+    Menu {
+        id: copyMenu
+
+        modal: true
+
+        MenuItem {
+            id: copy
+
+            text: qsTr("Copy")
+            onTriggered: Clipboard.setText(te.selectedText);
```

Comment:
https://doc.qt.io/qt-5/qml-qtquick-textedit.html#copy-method

Reply:
Who knew? :D

Reply:
It was I, the great Nico! ;p